src/main/webapp/js/instructorSearch.js

```diff
@@ -8,3 +8,13 @@ $(function() {
 	
 	$("div[id^=plainCommentText]").css("margin-left","15px");
 });
+
+/**
+ * Function that shows confirmation dialog for removing a student from search result
+ * @param studentName
```

Comment:
I understand that you copied this code from another js file but the comment is missing \@param courseId, and you can remove the empty \@return.